skbio/stats/ordination/_cutils.pyx

```diff
@@ -0,0 +1,165 @@
+# -----------------------------------------------------------------------------
+#  Copyright (c) 2013--, scikit-bio development team.
+#
+#  Distributed under the terms of the Modified BSD License.
+#
+#  The full license is in the file COPYING.txt, distributed with this software.
+# -----------------------------------------------------------------------------
+
+# distutils: extra_compile_args=-fopenmp
+# distutils: extra_link_args=-fopenmp
+
+import numpy as np
```

Comment:
should `cimport numpy as np` be used? 

https://cython.readthedocs.io/en/latest/src/tutorial/numpy.html#adding-types

Reply:
I don't think it is necessary anymore.

The old syntax was replaced by memoryviews:
https://cython.readthedocs.io/en/latest/src/userguide/numpy_tutorial.html#numpy-tutorial